gradle/graph.gradle

```diff
@@ -0,0 +1,29 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "com.vanniktech:gradle-dependency-graph-generator-plugin:0.3.0"
```

Comment:
For consistency we should add this dependency in the [dependencies.gradle](https://github.com/mapbox/mapbox-android-demo/blob/master/gradle/dependencies.gradle) file.

Reply:
This is the same approach `script-git-version.gradle` follows. If we're gonna change one script for consistency, we should change them all - feel free to cut a separate ticket.

Reply:
Follow up ticket 👉 https://github.com/mapbox/mapbox-android-demo/issues/676